nx-X11/programs/Xserver/dix/dispatch.c

```diff
@@ -111,7 +111,7 @@ int ProcInitialConnection();
 #ifdef XKB
 #define XKB_IN_SERVER
 #include "inputstr.h"
-#include <nx-X11/extensions/XKBsrv.h>
+#include <xkbsrv.h>
```

Comment:
I have now checked includes and adapted to upstream. While you are probably right in respect to quotes against brackets I prefer to stay as close as possible to upstream. Upstream is not really coherent in that respect, too...